src/compiler/crystal/tools/git.cr

```diff
@@ -0,0 +1,22 @@
+module Crystal::Git
+  class_property executable = "git"
+
+  # Tries to run git command with args.
+  # Yields block if exec fails or process status is not success.
+  def self.git_command(args, output : Process::Stdio = Process::Redirect::Close)
+    status = Process.run(executable, args, output: output)
+    return status.success?
```

Comment:
```suggestion
    status.success?
```

Reply:
Yes, return is implicit, but I'd prefer to be more explicit.